tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -7590,6 +7582,276 @@ def test_nvfp4_8gpus_mtp_custom_op(self, monkeypatch):
                           extra_evaluator_kwargs=self.EXTRA_EVALUATOR_KWARGS)
 
 
+class TestNemotronV3Ultra(LlmapiAccuracyTestHarness):
+    MODEL_NAME = "nvidia/Nemotron-Ultra-V3"
+    # No thinking mode for now.
+    EXTRA_EVALUATOR_KWARGS = dict(chat_template_kwargs=dict(
+        enable_thinking=False))
```

Ruff (0.15.14)

[warning] Mutable default value for class attribute

(RUF012)